src/riak_core_ring_manager.erl

```diff
@@ -406,7 +410,10 @@ handle_call({ring_trans, Fun, Args}, _From, State=#state{raw_ring=Ring}) ->
 handle_call({set_cluster_name, Name}, _From, State=#state{raw_ring=Ring}) ->
     NewRing = riak_core_ring:set_cluster_name(Ring, Name),
     State2 = prune_write_notify_ring(NewRing, State),
-    {reply, ok, State2}.
+    {reply, ok, State2};
+handle_call(is_stable_ring, _From, State) ->
```

Comment:
Nitpick: Please use spaces after commas in tuples here and below for consistency with surrounding code in this file.